recipe/meta.yaml

```
@@ -9,25 +8,31 @@ package:
 source:
   fn: {{ name }}-{{ version }}.tar.gz
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: {{ sha256 }}
+  sha256: 0c617dec19ed4f1ff6b247cd73855e949d87052d71900938b71f0cafd92f191b
 
 build:
-  noarch: python
   number: 0
-  script: {{ PYTHON }} -m pip install . -vv
+  script: {{ PYTHON }} -m pip install . -vv --no-build-isolation --no-deps
+  skip: true  # [py<39]
 
 requirements:
   host:
-    - python >=3.7
+    - python
     - pip
+    - flit-core <4
   run:
-    - python >=3.7
-    - marshmallow >=3.0.0
-    - packaging
+    - python
+    - marshmallow >=3.13.0,<5.0.0
```

```suggestion
    - marshmallow >=3.0.0
```
https://github.com/marshmallow-code/webargs/blob/8.7.0/pyproject.toml#L44

---

I saw [this](https://github.com/marshmallow-code/webargs/blob/dev/pyproject.toml#L45)

---

That's my bad! I was on the `dev` branch.

---

Gotcha. That's the dev branch. We I think we want the 8.7.0 branch?

---

> That's my bad! I was on the `dev` branch.

Yeah